client/src/components/screens/PatientCreation/components/FormModal/index.tsx

```diff
@@ -101,6 +100,22 @@ function validateForm(formValues: any) {
   });
 }
 
+type MrnData ={
+  mrn: string;
+  hospital: string;
+}
+
+const extractMrnData = (patient: Patient) : MrnData | undefined => {
+  const identifier = patient.identifier.find((id) => get(id, 'type.coding[0].code') === 'MR');
+  if (identifier == null) {
+    return undefined;
+  }
+  return {
+    mrn: identifier.value,
```

Comment:
does your linter force you to do that? undefined is default if you only return your object on condition negation.

Reply:
I just prefer to be explicit about what the function returns